### ISSUE TITLE
New workflow syntax?

### ISSUE THREAD

**Issue by aequasi** [actions/zeit-now#10]

**Comment by tedmiston** [comment]
I have the hunch with v2 this may be deprecated in favor of using `actions/setup-node` and just installing now yourself.  That's what I'm doing at least.

**Comment by byCedric** [comment]
Yeah, I would recommend the same thing. I'm waiting on the caching-across-jobs feature, only with that feature you might get some performance benefits over installing it yourself.

For now, I would recommend add now as a dev dependency and just run `npx now` (or what ever that is in yarn) 😁
